spaces/domains.json

```
@@ -961,5 +961,6 @@
   "governance.topia.gg": "topia-governance.eth",
   "gov.piteas.io": "piteas.eth",
   "vote.aitech.io": "aitechio.eth",
-  "vote.orca.monster": "skinnyorca.eth"
+  "vote.orca.monster": "skinnyorca.eth",
+  "vote.pillardao.org": "plrdao.eth",
```

You will need to remove the comma at the end of the line of your sub-domain.
```suggestion
  "vote.pillardao.org": "plrdao.eth"
```

---

hello, Thank you.
Could you please re-review.